renovate.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+      "config:base"
+    ],
+    "prHourlyLimit": 4,
+    "baseBranches": ["main", "release-2.10", "release-2.9"],
```

Comment:
Does this mean we have to update this each time we do a release? Then please add it to the release instructions at `RELEASE.md`

Reply:
Yes we do at the moment.